rps.js

```diff
@@ -0,0 +1,13 @@
+var Player1 = prompt ("Player1: rock, paper, scissors?");
```

Comment:
The program doesn't work. It doesn't have a function, also player 2 should a random choice by the computer.